lib/internal/errors.js

```diff
@@ -164,7 +164,7 @@ E('ERR_SOCKET_BAD_PORT', 'Port should be > 0 and < 65536');
 E('ERR_SOCKET_DGRAM_NOT_RUNNING', 'Not running');
 E('ERR_V8BREAKITERATOR', 'full ICU data not installed. ' +
   'See https://github.com/nodejs/node/wiki/Intl');
-E('NULL_REJECTION', 'Promise was rejected with "null"');
+E('FALSE_REJECTION', 'Promise was rejected with falsy value');
```

Comment:
The message change is okay, but I liked `NULL_REJECTION` better as a code; `FALSE_REJECTION` sounds like “this is not really a rejection”, even if it totally is one.